src/SprykerEco/Zed/Braintree/Business/Order/Saver.php

```diff
@@ -85,4 +84,25 @@ protected function savePaymentForOrderItems($orderItemTransfers, $idPayment)
             $paymentOrderItemEntity->save();
         }
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Generated\Shared\Transfer\SaveOrderTransfer $saveOrderTransfer
+     *
+     * @return void
+     */
+    public function updateOrderPayment(QuoteTransfer $quoteTransfer, SaveOrderTransfer $saveOrderTransfer): void
+    {
+        $paymentTransfer = $quoteTransfer->getPayment();
+        if ($paymentTransfer->getPaymentProvider() === BraintreeConfig::PROVIDER_NAME) {
+            $idSalesOrder = $saveOrderTransfer->getIdSalesOrder();
+            $paymentEntity = SpyPaymentBraintreeQuery::create()
```

Comment:
Please move handling of propel entities to the entity manager.

Reply:
Updated.